projects/canopy/src/lib/forms/radio/radio-button.component.html

```diff
@@ -13,3 +13,4 @@
 <label [attr.for]="id" class="lg-radio-button__label">
   <ng-content></ng-content>
 </label>
+<ng-content select="lg-hint"></ng-content>
```

Comment:
Can you modify the story so that we can have a preview of how it looks with the `lg-hint` please?

Reply:
Added an `<lg-hint>` tag to one of the radio buttons 😉